src/Umbraco.Web/Unversion/UnversionConfig.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Web;
+using System.Xml;
+using Umbraco.Core.Logging;
+
+namespace Umbraco.Web.Unversion
+{
+    public class UnversionConfig : IUnversionConfig
+    {
+        public const string AllDocumentTypesKey = "$_ALL";
+        public IDictionary<string, List<UnversionConfigEntry>> ConfigEntries { get; set; }
+
+        private ILogger _logger;
+
+        public UnversionConfig(ILogger logger)
+        {
+            _logger = logger;
+
+            ConfigEntries = new Dictionary<string, List<UnversionConfigEntry>>();
+
+            try
+            {
+                var appPath = HttpRuntime.AppDomainAppPath;
```

Comment:
Please do not use this. Use IOHelper.MapPath to get paths which is unit testable, this is not.